ogcapi-features/openapi.yaml

```diff
@@ -97,7 +97,7 @@ paths:
         - Features
       summary: the feature collections in the dataset
       description: |-
-        A body of Feature Collections that belong or are used together with additional links.
```

Comment:
I'd rather stick to the original. In OGC API Features they don't use the appended "object" to entities.